DesktopUI2/DesktopUI2/ViewModels/HomeViewModel.cs

```diff
@@ -114,7 +115,7 @@ internal void UpdateSelectedStream()
   {
     try
     {
-      if (_selectedSavedStream != null)
```

Comment:
Took the freedom to add this @clairekuang @AlanRynne - we do not need to update a Stream while it's currently sending or receiving.

Maybe a debouncer would be needed on the event-triggering side too.